File: 2026-1/sesion-de-clase-04/java/seguro-vehicular/pe/edu/softseguros/seguros/SeguroPlata.java
package pe.edu.softseguros.seguros;

import pe.edu.softseguros.modelo.Vehiculo;

public class SeguroPlata extends SeguroBronce {

    public SeguroPlata(Vehiculo vehiculo) {
        super(vehiculo);
    }

    @Override
    public double CalcularCosto() {
        return super.CalcularCosto() + 100.00;
    }

    @Override
    public String Descripcion() {
        return super.Descripcion() + " + asistencia vial";
    }
}

File: 2026-1/sesion-de-clase-05/dotnet/seguroslib/Seguros/SeguroPlata.cs
namespace Seguros;

using Modelo;

public class SeguroPlata : SeguroBronce
{
    public SeguroPlata(Vehiculo vehiculo) : base(vehiculo)
    {
    }

    public override double CalcularCosto()
    {
        return base.CalcularCosto() + 100.00;
    }

    public override string Descripcion()
    {
        return base.Descripcion() + " + asistencia vial";
    }
}
